### ISSUE TITLE
Custom Health woes

### ISSUE THREAD

**Issue by BenFRC5147** [Exiled-Team/Common-Utils#126]
**Describe the bug**
For some reason, the custom health for the Captain is applied, but it seems like their maximum health is set to 250, as when I heal via admin command their health can go all the way up that high, but the SCP is set to that HP and doesn't have their Max HP modified, as when I heal them their health doesn't go up to 4500, but reverts to 4000.

**To Reproduce**
Set health of Captain higher than normal
spawn as captain
see that health bar isn't full, and can heal past the set health of 150

Set health of scp173 higher than normal
spawn as scp173 
health bar unchanged, doesn't move until below standard health
when force healed, reset to base HP

**Expected behavior**
Max health is set for both and to the expected value

**Server logs**
Please include a pastebin of your localadmin log file (or both MA_log and SCP_log files if you use MultiAdmin) from the time in which the bug occured
https://pastebin.com/PdDaB2rE

**Copy of current CU config section**
  # A list of roles and what their default starting health should be.
  health_values:
    Scp173: 4500
    NtfCaptain: 150

**EXILED Version ("latest" is not a version):**
![image](https://github.com/Exiled-Team/Common-Utils/assets/13715542/fee6008d-3a50-4cf7-b803-c7d48607dc8e)

**Results of `show plugins` command in console:**
```
                                 Exiled.Events:
                                        - Author: Exiled Team
                                        - Version: 8.3.4.0
                                        - Required Exiled Version: 8.3.4.0
                                        - Prefix: exiled_events
                                        - Priority: Highest
                                 Common Utilities:
                                        - Author: Joker119
                                        - Version: 0.0.0.0
                                        - Required Exiled Version: 8.0.0
                                        - Prefix: CommonUtilities
                                        - Priority: Higher
                                 Gambling With Quarters:
                                        - Author: PatPeter and Babyboucher20
                                        - Version: 4.10.4.1
                                        - Required Exiled Version: 5.1.3
                                        - Prefix: gambling_with_quarters
                                        - Priority: Medium
                                 Exiled.Permissions:
                                        - Author: Exiled Team
                                        - Version: 8.3.4.0
                                        - Required Exiled Version: 8.3.4.0
                                        - Prefix: exiled_permissions
                                        - Priority: Medium
                                 Exiled.CustomItems:
                                        - Author: Exiled Team
                                        - Version: 8.3.4.0
                                        - Required Exiled Version: 8.3.4.0
                                        - Prefix: exiled_custom_items
                                        - Priority: `Medium`
                                 scp-294:
                                        - Author: batatas-fritas
                                        - Version: 2.0.1
                                        - Required Exiled Version: 8.0.0
                                        - Prefix: scp294
                                        - Priority: Medium
                                 Admin Tools:
                                        - Author: Originally by Joker119. Modified by Benjamin
                                        - Version: 8.0.0.0
                                        - Required Exiled Version: 8.0.0
                                        - Prefix: AT
                                        - Priority: Medium
                                 CedMod:
                                        - Author: ced777ric#8321
                                        - Version: 3.4.15
                                        - Required Exiled Version: 8.0.0
                                        - Prefix: cm
                                        - Priority: Medium
                                 MapEditorReborn:
                                        - Author: Michal78900
                                        - Version: 3.0.2
                                        - Required Exiled Version: 8.0.0
                                        - Prefix: map_editor_reborn
                                        - Priority: Medium
                                 Exiled.CustomRoles:
                                        - Author: Exiled Team
                                        - Version: 8.3.4.0
                                        - Required Exiled Version: 8.3.4.0
                                        - Prefix: exiled_custom_roles
                                        - Priority: Medium
                                 Exiled.CreditTags:
                                        - Disabled
                                        - Author: Babyboucher20 & iRebbok & Exiled Team
                                        - Version: 8.3.4.0
                                        - Required Exiled Version: 8.3.4.0
                                        - Prefix: exiled_credits
                                        - Priority: Medium
                                 Unigamia Patreon Rewards:
                                        - Author: PatPeter and Babyboucher20
                                        - Version: 6.0.0
                                        - Required Exiled Version: 5.1.3
                                        - Prefix: upr
                                        - Priority: Medium
                                 RespawnTimer:
                                        - Author: Michal78900
                                        - Version: 4.0.2
                                        - Required Exiled Version: 7.0.5
                                        - Prefix: respawn_timer
                                        - Priority: Lowest
```

**Additional context**

**Comment by BenFRC5147** [comment]
Also I have health gain on kill set to default, which has 0 for 173, and when 173 kills somebody his health resets to base game level down from the buffed level I have my config set to

**Comment by BenFRC5147** [comment]
Fixed by Exiled 8.3.6
